### PR TITLE
Platform | Fix review page accordions

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > - Fix accordion edit button alignment
  > - Fix accordion error styling
  > - Removed duplicate alert
  > - Fix contact info button (previously full width)
  > - Fix contact info edit focus management
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits decision review
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#64149](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64149)

## Testing done

- _Describe what the old behavior was prior to the change_
  > Misalignment & broken styling of review page accordion
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
  > Updated effected unit tests
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Page errors | Screenshot 2023-12-05 at 9 26 12 AM | Screenshot 2023-12-05 at 9 26 55 AM |
| Contact info | Screenshot 2023-12-05 at 9 26 02 AM | Screenshot 2023-12-05 at 9 26 46 AM |

## What areas of the site does it impact?

All forms 

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/platform/forms-system/src/js/review/ObjectField.jsx [hunk 1]
```diff
@@ -187,7 +187,7 @@ class ObjectField extends React.Component {
                   {title}
                 </h4>
               )}
-            <div className="vads-u-width--full vads-u-display--flex vads-u-justify-content--flex-end">
```
Comment (Mottie) [inline src/platform/forms-system/src/js/review/ObjectField.jsx:190]: `vads-u-width--full` was forcing the edit button to a new line

File: src/platform/forms-system/src/js/components/ContactInfoReview.jsx [hunk 2]
```diff
@@ -30,13 +32,12 @@ const ContactInfoReview = ({ data, editPage, content, keys }) => {
...
-        editRef?.current
-      ) {
+      if (getReturnState() === 'true,' && editRef?.current) {
         // focus on edit button _after_ editing and returning
-        window.sessionStorage.removeItem(CONTACT_EDIT);
-        setTimeout(() => focusElement(editRef.current));
+        clearReturnState();
+        setTimeout(() =>
+          focusElement('button', {}, editRef.current.shadowRoot),
+        );
```
Comment (adamwhitlock1) [inline src/platform/forms-system/src/js/components/ContactInfoReview.jsx:40]: Is this timeout just to push this focusElement to the end of the event loop/task queue?

**Review by adamwhitlock1 — APPROVED** [review]
LGTM. I think the removal of the reliance of the session storage value is a good call.

🚀

File: src/platform/forms-system/src/js/components/ContactInfoReview.jsx (hunk 2 above)
Comment (Mottie) [reply src/platform/forms-system/src/js/components/ContactInfoReview.jsx:40]: > I think the removal of the reliance of the session storage value is a good call.
It didn't really get removed 😞 - the code was switched to use functions that manipulate session storage

**Review by powellkerry — APPROVED** [review]

**Pull request merged**